src/styles/schema-styles.js

```diff
@@ -56,7 +56,7 @@ export default css`
 .inte, .integer, .numb, .number, .int6, .int64, .int3, .int32, .floa, .float, .doub, .double, .deci, .decimal, .blue { color:var(--blue); }
 .null { color:var(--red); }
 .bool, .boolean { color:var(--orange); }
-.enum, .cons, .const { color:var(--yellow); }
+[class$="enum"], .cons, .const { color:var(--yellow); }
```

Comment:
There's a bug with the enum css type, we should fix it there instead of here, I'll add a comment with the suggested fix.

Reply:
```suggestion
.enum, .cons, .const { color:var(--yellow); }
```